primitives/manta/src/assets.rs

```diff
@@ -157,9 +136,29 @@ where
     >;
 }
 
+/// Bound for SBT Metadata Size (Could make generic over runtimes, but simpler to just hardcode)
+pub type SbtBound = ConstU32<100>;
+
+/// Metadata for each Asset Type
+///
+/// Currently supports SBT (Soul Bound Token) and Fungible Tokens
+#[derive(Clone, Debug, Decode, Encode, Eq, PartialEq, TypeInfo, MaxEncodedLen)]
+pub enum AssetMetadata<Balance> {
```

Comment:
Do we need migration for already existed AssetMetadata?

Reply:
Yes we do. 

I think follow up PR, will make it better to review to keep diff smaller

Reply:
From what I understood we're not developing this on Calamari, so no need.

Reply:
Buut actually this still needs to change in Calamari...

Reply:
Yeah  I think this is the best way to accumulate no tech debt. Its fine if some variants are not used on both Calamari/Manta